src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -381,7 +381,7 @@ private void RefreshExpandedDisplay(Action refreshDisplay)
                 () =>
                 {
                     newViewModel = nodeViewModel.DynamoViewModel.WatchHandler.GenerateWatchViewModelForData(
-                        mirrorData, null, string.Empty, false);
+                        mirrorData, null, nodeViewModel.NodeModel.AstIdentifierForPreview.Name, false);
```

Comment:
with empty tag here, HelixWatch3DViewModel will get `path` as `:0:1` rather than `var_guid:0:1`, so it has been no way to determine which node is requested to add labels